src/main/java/org/broadinstitute/hellbender/tools/funcotator/mafOutput/MafOutputRenderer.java

```diff
@@ -220,6 +220,10 @@ public MafOutputRenderer(final Path outputFilePath,
 
     @Override
     public void close() {
+        if (!hasWrittenHeader) {
+            final LinkedHashMap<String, String> mafCompliantOutputMap = createMafCompliantOutputMap(Allele.create("AT"), Collections.emptyList());
```

Comment:
What is `"AT"`?

Reply:
Put in some clarifying comments.

Done.